manifests/init.pp

```diff
@@ -249,8 +249,7 @@
     default => '--no-sshd',
   }
 
-  class {'::easy_ipa::validate_params':}
-  -> class {'::easy_ipa::install':}
-
+  require easy_ipa::validate_params
```

Comment:
Interestingly I've never used the "require" function. Looks better now, I don't like the look of the ordering arrows anyways :smile:.